static/app/components/platformPicker.tsx

```diff
@@ -108,7 +109,11 @@ class PlatformPicker extends React.Component<Props, State> {
               <ListLink
                 key={id}
                 onClick={(e: React.MouseEvent) => {
-                  analytics('platformpicker.select_tab', {category: id});
+                  trackAdvancedAnalyticsEvent(
+                    'growth.platform_category',
+                    {category},
+                    null
```

Comment:
@roggenkemper we should pass down the `organization` so we can use it in analytics (but it can be an optional param).